src/System.Private.Xml/src/System/Xml/Serialization/XmlSerializer.cs

```diff
@@ -121,14 +121,22 @@ public abstract class XmlSerializerImplementation
     /// </devdoc>
     public class XmlSerializer
     {
+#if uapaot
```

Comment:
I assumed if a type was not in the contract then it shouldn't be public in the implementation assembly for netcoreapp. If that's not a problem, I'm ok to remove the ifdef.